cmd/minishift/cmd/start.go

```diff
@@ -231,6 +231,7 @@ func postClusterUp(machineName string, ip string, port int, routingSuffix string
 		atexit.Exit(1)
 	}
 
+	configurePeristentVolumes(sshCommander)
```

Comment:
this was my previous concern (as we now handle 100 directories), but this is an arbitrary number. It could even be this is just 10... and rather JUST ten. But is this configurable from the command line for `cluster up`?